velox/exec/WindowPartition.h

```diff
@@ -40,11 +41,37 @@ class WindowPartition {
       const std::vector<std::pair<column_index_t, core::SortOrder>>&
           sortKeyInfo);
 
+  virtual ~WindowPartition() = default;
+
   /// Returns the number of rows in the current WindowPartition.
-  vector_size_t numRows() const {
+  virtual vector_size_t numRows() const {
     return partition_.size();
   }
 
+  virtual vector_size_t offsetInPartition() const {
```

Comment:
Please can you add documentation for these methods. Would make it easier to review the code.

Reply:
@JkSelf +1. Please, document new APIs.